kp_test.go

```diff
@@ -1626,6 +1626,49 @@ func TestSetAndGetInstancePolicies(t *testing.T) {
 	assert.True(t, *(ap.PolicyData.Enabled))
 	assert.Equal(t, ap.PolicyData.Attributes.AllowedNetwork, "public-and-private")
 
+	// Set and get allowed ip instance policy
+	allowedIPPolicy := []byte(`{
+		"metadata": {
+			"collectionType": "application/vnd.ibm.kms.policy+json",
+			"collectionTotal": 1
+		},
+		"resources": [{
+			"creationDate": "2020-09-01T17:28:28Z",
+			"createdBy": "IBMid-50BE1MTM26",
+			"updatedBy": "IBMid-50BE1MTM26",
+			"lastUpdated": "2020-09-02T17:20:21Z",
+			"policy_type": "allowedIP",
+			"policy_data": {
+				"enabled": true,
+				"attributes": {
+					"allowed_ip": ["47.220.195.239/32", "12.184.193.84/30"]
```

Comment:
We should avoid referencing real ip addresses. I've provided some documentation ones
```suggestion
					"allowed_ip": ["198.51.100.0/32", "2001:db8::/128"]
```